src/controller/tests/BUILD.gn

```diff
@@ -40,7 +40,10 @@ chip_test_suite("tests") {
 
     # Not supported on efr32.
     if (chip_device_platform != "efr32") {
-      test_sources += [ "TestCommissioningWindowOpener.cpp" ]
+      test_sources += [
+        "TestAutoCommissioner.cpp",
```

Comment:
why is autom commissioner not supported on efr32? Can we update comments with reasons (like no RAM or flash to low or compile error or something else)?